tests/spdx/writer/tagvalue/test_file_writer.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from unittest.mock import MagicMock, call, mock_open, patch
+
+from spdx.writer.tagvalue.file_writer import write_file
+from tests.spdx.fixtures import file_fixture
+
+
+def test_file_writer():
+    spdx_file = file_fixture()
+
+    mock: MagicMock = mock_open()
+    with patch(f"{__name__}.open", mock, create=True):
+        with open("foo", "w") as file:
+            write_file(spdx_file, file)
+
+    mock.assert_called_once_with("foo", "w")
+    handle = mock()
+    handle.write.assert_has_calls(
+        [
+            call("## File Information\n"),
+            call(f"FileName: {spdx_file.name}\n"),
+            call(f"SPDXID: {spdx_file.spdx_id}\n"),
+            call(f"FileType: {spdx_file.file_types[0].name}\n"),
+            call("FileChecksum: SHA1: 71c4025dd9897b364f3ebbb42c484ff43d00791c\n"),
+            call("LicenseConcluded: MIT AND GPL-2.0-only\n"),
+            call("LicenseInfoInFile: MIT\n"),
+            call("LicenseInfoInFile: GPL-2.0-only\n"),
+            call("LicenseInfoInFile: NOASSERTION\n"),
```

Comment:
You missed a few here? (package and snippet have their license info referenced, too)